tests/unit/components/Records/Search/Header/FilterChips.spec.js

```diff
@@ -39,12 +39,8 @@ describe("FilterChips.vue", () => {
         wrapper.vm.$router.push = push;
         await wrapper.vm.removeParam("publications", "pub2");
         expect($router.push).toHaveBeenCalledTimes(1);
-        let query = wrapper.vm.buildNewQuery("grants", "ABCDEF");
-        expect(query).toStrictEqual({page: 1, publications: 'pub1,pub2,pub3'});
-        query = wrapper.vm.buildNewQuery("publications", "pub4");
-        expect(query).toStrictEqual({page: 1, grants: 'ABCDEF', publications: 'pub1,pub2,pub3'});
-        await wrapper.vm.removeParam("publications", "pub3");
-        expect($router.push).toHaveBeenCalledTimes(1);
```

Comment:
Why did you remove these tests?

Reply:
by confusion! unfortunately they were taken out but are put back in 4.1